mwaz/flask_api
# Ft mashmallow validation 154065039

## What does this PR do?

Separate Logic from user form validations

##  Description of Task to be completed?
Separation of logic and validation in auth, category, and recipes functions
using marshmallow validation to raise errors in the event of invalid data input

##  How can this be manually tested?
Use postman or the [online documentatio](yummy-recipies.herokuapp.com)n to check the errors being raised by 
marshmallow validator

##  What are the relevant pivotal tracker stories?
[#154065039](https://www.pivotaltracker.com/story/show/#154065039)
